app/config.gradle

```diff
@@ -1,26 +1,25 @@
 ext {
-    compileSdkVersion = 25
-    buildToolsVersion = "26.0.2"
```

Comment:
I have used version 3.1.2 of Android plugin for Gradle 4.7. Hence it is not required.

Reply:
Nice